docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1016,6 +1016,104 @@ table my_ipv4_lkup {
   ...
 }
 ~ End p4example
+### Custom Annotations in P4Info
+P4 allows flexible annotations[@P4Annotations]. Users may add their own annotations as long as they obey the language syntax. To facilitate consumption of user-defined annotations, P4Info has an `annotation_map` for any element which can accept such annotations, &ie; the `PkgInfo`, `Preamble`, `MatchField`, `ActionRef`, `Action` and `ControllerPacketMetadata` messages. The `annotation_map` contains a suitable parsed representation for the different types of annotation bodies, namely: `expression_list` and `kv_map`. Empty annotation bodies are signified by an empty `value` field  in the `annotation_map` entry. 
```

Comment:
looks like I forgot about p4types.proto...

so me may need to rephrase:

> PkgInfo, ..., ControllerMetadata and the type specification messages in p4types.proto

Reply:
Fixed, thanks.